fineract-provider/src/main/java/org/apache/fineract/portfolio/delinquency/api/DelinquencyApiResourceSwagger.java

```diff
@@ -133,4 +133,19 @@ private DeleteDelinquencyBucketResponse() {}
         public Integer resourceId;
     }
 
+    @Schema(description = "GetDelinquencyTagHistoryResponse")
+    public static final class GetDelinquencyTagHistoryResponse {
+
+        private GetDelinquencyTagHistoryResponse() {}
+
+        @Schema(example = "1")
+        public Integer id;
+        @Schema(example = "Delinquent 1")
+        public String classification;
+        @Schema(example = "1")
+        public Long minimumAgeDays;
```

Comment:
It should contains the date when it got added and when it got lifted

Reply:
Done!